### PR TITLE
feat(docs): add kind - ingress instructions to deploy model registry

### PR DESCRIPTION
**Pull request by Al-Pragliola**
## Description

I have added a small guide to deploy model registry on a kind cluster using an ingress instead of port-forward, hoping people will find it useful for local testing

## How Has This Been Tested?

followed locally the steps described in the guide

## Merge criteria:

- All the commits have been [_signed-off_](https://github.com/kubeflow/community/tree/master/dco-signoff-hook#signing-off-commits)  (To pass the `DCO` check)

- [x] The commits have meaningful messages; the author will squash them after approval or in case of manual merges will ask to merge with squash.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work.
- [x] Code changes follow the [kubeflow contribution guidelines](https://www.kubeflow.org/docs/about/contributing/).

### CODE REVIEW

**Review by tarilabs — COMMENTED** [review]
Thank you @Al-Pragliola ; some small considerations below: wdyt?
 
Overall,
/lgtm

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -61,6 +61,34 @@ to your PATH from your bashrc like:
...
+
+### Port-forwarding guide
+
+Create a Kind cluster with the following command:
+
+```sh
+kind create cluster
+```
+
+and then follow the steps from this link [Installation guide](https://www.kubeflow.org/docs/components/model-registry/installation/#standalone-installation) to set up the port-forwarding and deploy the Model Registry on the cluster.
```
Comment (tarilabs) [inline CONTRIBUTING.md:86]:
```suggestion
and then follow the steps from the [Installation guide](https://www.kubeflow.org/docs/components/model-registry/installation/#standalone-installation) on the Kubeflow website, to set up the port-forwarding and deploy the Model Registry on the cluster.
```

wdyt?

**Review by tarilabs — COMMENTED** [review]
/lgtm

File: CONTRIBUTING.md (hunk 1 above)
Comment (Al-Pragliola) [reply CONTRIBUTING.md]: 👍 definitely better

**Review by tarilabs — APPROVED** [review]
awesome, thank you @Al-Pragliola , this will help to showcase options for local testing to contributors in an easy/easier way!

/lgtm
/approve

**Pull request merged**